app/kumactl/cmd/get/get_fault_injections.go

```diff
@@ -45,17 +45,17 @@ func newGetFaultInjectionsCmd(pctx *getContext) *cobra.Command {
 	return cmd
 }
 
-func printFaultInjection(faultInjections *mesh.FaultInjectionResourceList, out io.Writer) error {
+func printFaultInjection(faultInjections []*mesh.FaultInjectionResource, out io.Writer) error {
```

Comment:
I missed this when this func was added. Can we rename it to `printFaultInjections`?